text/metrics/0001-naming-conventions.md

```diff
@@ -0,0 +1,137 @@
+# Metric naming conventions
```

Comment:
Should this be "Metric instrument naming conventions" since the thing that instrumentation will do is create instruments, not metrics?